examples/graphql-idl-support/graphql_idl_support.bal

```diff
@@ -0,0 +1,10 @@
+import ballerina/io;
+client "./graphql.config.yaml" as foo;
+
+public function main() returns error? {
+    foo:client graphqlClient = check new (serviceUrl = "https://countries.trevorblades.com/");
```

Comment:
shall we use `trevorblades` or a similar meaningful alias for the imported client instead of using `foo`?